docs/src/analysis/sensitivity.md

```diff
@@ -416,7 +417,7 @@ the adjoint sensitivity of some discontinuous functional of the solution. One
 canonical function is the L2 loss against some data points, that is:
 
 ```math
-L(u,p,t)=\sum_{i=1}^{n}\Vert\tilde{u}(t_{i})-u(t_{i},p)\Vert^{2}
```

Comment:
it could be explicitly time dependent IIRC

Reply:
But is does not, t is gone after the summation over time. The same was with G. You have the functional G(u,p) which operates on the function t->u(t). Both functionals are continuous as mappings from u -> R, thus "discontinous functional" is better rephrased (like "discrete functional", "functional evaluating a discrete points etc.).